### PR TITLE
plandek-deployment 1.0.0

### PR DESCRIPTION
**Pull request by kdybicz**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

**New Step**
Thank you for the new Step share! The CI check might will fail due to our extended validation engine. Nothing to worry about yet, we will get back to you shortly.

### CODE REVIEW

File: steps/plandek-deployment/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+title: Plandek - deployment
+summary: |
+  Enables you to track and visualize build and deployment events within Plandek's metrics and dashboards.
+description: |
+  Wrapper implementing the Plandek /deployment POST API. Used to complete a deployment within Plandek for a given release cycle.
+  More details can be found here: https://api.plandek.com/deployments/v1/ui/
+website: https://github.com/secretescapes/bitrise-step-plandek-deployment
```
Comment (lpusok) [inline steps/plandek-deployment/1.0.0/step.yml:7]: [recommendation] Please replace with https://plandek.com/.

File: steps/plandek-deployment/1.0.0/step.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    entry_file: step.sh
+is_always_run: true
+is_skippable: true
+inputs:
+- api_token: null
+  opts:
+    description: |
+      Plandek API token.
+    is_required: true
+    is_sensitive: true
```
Comment (lpusok) [inline steps/plandek-deployment/1.0.0/step.yml:27]: Please add `is_expand: true`, as it is required in case is_sensitive is true.

**Review by lpusok — CHANGES_REQUESTED** [review]
Thanks, left a few comments. When fixed please re-share the Step.

File: steps/plandek-deployment/1.0.0/step.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  opts:
+    description: |
+      Unique identifier for your instance.
+    is_required: true
+    title: Client key
+- opts:
+    description: "Typically corresponds to the name of your pipeline, workflow, repo,
+      or service under which builds are compiled.\nIf you use the same name pipelines
+      across your organization, we recommend appending additional information to\nyour
+      pipeline name (e.g. repo name) to make it easier to understand your data. \n"
```
Comment (koral--) [inline steps/plandek-deployment/1.0.0/step.yml:39]:
```suggestion
      pipeline name (e.g. repo name) to make it easier to understand your data."
```

File: steps/plandek-deployment/1.0.0/step.yml (hunk 1 above)
Comment (kdybicz) [reply steps/plandek-deployment/1.0.0/step.yml:7]: @lpusok what do you mean, which link should I replace with https://plandek.com/ ?

Comment (lpusok) [reply steps/plandek-deployment/1.0.0/step.yml:7]: Yes.

File: steps/plandek-deployment/1.0.0/step.yml (hunk 2 above)
Comment (kdybicz) [reply steps/plandek-deployment/1.0.0/step.yml:27]: @lpusok fyi: from what I understand `is_expand: true` is not needed as it's default value when `is_sensitive: true` -> https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#sensitive-inputs but I don't mind leaving it as is.

Comment (lpusok) [reply steps/plandek-deployment/1.0.0/step.yml:27]: You are right, it can be omitted.

**Comment by kdybicz** [comment]
Thanks for the review, I'm gonna follow the https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/sharing-steps-with-all-bitrise-users.html#fixing-issues-in-a-steplib-pull-request for updates.

**Pull request closed without merging**